modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -212,20 +213,19 @@ public String toDefaultValue(Schema p) {
         }
 
         // convert datetime and date enums if they exist
-        DateFormat iso8601Date = new SimpleDateFormat("yyyy-MM-dd", Locale.ROOT);
-        DateFormat iso8601DateTime = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX", Locale.ROOT);
-        TimeZone utc = TimeZone.getTimeZone("UTC");
-        iso8601Date.setTimeZone(utc);
-        iso8601DateTime.setTimeZone(utc);
+        DateTimeFormatter iso8601Date = DateTimeFormatter.ofPattern("yyyy-MM-dd", Locale.ROOT);
```

Comment:
I'd recommend using `DateTimeFormatter.ISO_LOCAL_DATE` which resolves values strictly. That is, it will fail for invalid dates.

Reply:
Done!